### PR TITLE
GP2-1517: Port homepage model from V1 to V2 

### PR DESCRIPTION
**Pull request by stevejalim**
This changes brings over the Domestic homepage model into Magna.

Note that we ALREADY have a minimal homepage model in Magna, for use in the MVP and private beta as a landing page (`domestic.DomesticHomePage`). 

This new/ported model `domestic.models.GreatDomesticHomePage` has (almost) all the functionality of the V1 domestic homepage model and - eventually - will be swapped in to be the main root page of Magna.

Things the ported model does not have that V1 has:

* support for the EU Exit 'chevron' block - now no longer needed but we can reinstate something at short notice in the future if the idea of a 'chevron' is needed
* the dynamic selection of the markets - this has been split out to a separate ticket

### Workflow

- [X] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-1517
- [X] Jira ticket has the correct status.
- [X] [Changelog](CHANGELOG.md) entry added.

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: core/templates/components/informative_banner.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="container">
+    <div class="informative-banner">
+        <div>
+            {% if banner_label %}
+                <h2>{{ banner_label }}</h2>
+            {% endif %}
+            {% if banner_content %}
```
Comment (bobmeredith) [inline core/templates/components/informative_banner.html:7]: Wondering if the conditional is needed here. If banner_content is None, nothing will be rendered.

Comment (stevejalim) [reply core/templates/components/informative_banner.html:7]: True!  

@lewis-coulson-dit -- I think the conditional is fine to remove, yep? Would you mind?

Comment (lewis-coulson-dit) [reply core/templates/components/informative_banner.html]: Good spot. sorted

**Review by lewis-coulson-dit — APPROVED** [review]

**Review by Rambowalsh — COMMENTED** [review]
All fine

**Review by Rambowalsh — COMMENTED** [review]
No Problems

**Pull request merged**